src/components/multi-action-button/multi-action-button.component.js

```diff
@@ -71,6 +71,12 @@ class MultiActionButton extends SplitButton {
 }
 
 MultiActionButton.propTypes = {
+  /** The additional button to display. */
```

Comment:
This should have the margin props in the `propTypes` here, see https://github.com/Sage/carbon/blob/master/src/components/icon-button/icon-button.component.js#L42 for an example

Reply:
and what you've done in split button

Reply:
Also the margin props aren't actually getting passed to `StyledMultiActionButton` anywhere

Reply:
The prop types come from `SplitButton` which MultiActionButton extends, if they are passed to the `StyledMultiActionButton` then they are set twice. For now I think this is the right approach and when they are refactored to remove the extension we can then spread them on to the root